### DISCUSSION TITLE
Running on Windows 10

### DISCUSSION THREAD

**Discussion by kokozweeb** [acheong08/ChatGPT#222] [Q&A] [answered]
Is there a way to run revChatGPT on Windows 10?

**Comment by acheong08** [comment, marked as answer]
Yes. Just make sure to use 64bit python 3.10

**Reply by kokozweeb** [reply]
Can I also use Python 3.11.1 - Dec. 6, 2022?

**Reply by acheong08** [reply]
You can try. I haven't tested but it should work

**Reply by kokozweeb** [reply]
![image 1](https://user-images.githubusercontent.com/21144281/206883901-3afc8555-05ba-47b8-abc5-fc9a0b097d3a.png)

**Reply by acheong08** [reply]
Follow the instructions https://github.com/acheong08/ChatGPT/wiki/Setup

**Reply by acheong08** [reply]
`pip3 install revChatGPT --upgrade`

**Reply by kokozweeb** [reply]
> `pip3 install revChatGPT --upgrade`

isn't that a Linux console command?

**Reply by acheong08** [reply]
It's the same for windows if you have python installed

**Reply by kokozweeb** [reply]
![image_2022-12-11_040345558](https://user-images.githubusercontent.com/21144281/206884365-1c4e6520-31b7-45cf-bc59-10fc31ab0d58.png)

**Reply by acheong08** [reply]
If it's not pip3, it's `python3 -m pip`

**Reply by acheong08** [reply]
Or `python -m pip` if you have that set as python3

**Reply by kokozweeb** [reply]
Do I need to add Python to environment variable path?

**Reply by acheong08** [reply]
How is your python configured? It should work by default `python -m pip install revChatGPT`

**Reply by kokozweeb** [reply]
![image_2022-12-11_040633184](https://user-images.githubusercontent.com/21144281/206884424-30938072-ec56-43e8-a891-7917555fc4d6.png)
I think I need to set the environment variable path

**Reply by acheong08** [reply]
yes

**Reply by kokozweeb** [reply]
![image_2022-12-11_041034949](https://user-images.githubusercontent.com/21144281/206884507-0cef4c33-a015-4e3d-95da-81104c6247b7.png)
Alright I think I'm getting the hang of how Python works

**Reply by kokozweeb** [reply]
How do I interact with ChatGPT AI? :S

**Reply by acheong08** [reply]
https://github.com/acheong08/ChatGPT/wiki/CLI-use

Please read the wiki
